### DISCUSSION TITLE
Can we add the scram-sha-256 user authrization on kafka operator

### DISCUSSION THREAD

**Discussion by yinkp666** [strimzi/strimzi-kafka-operator#6572] [Q&A] [answered]
apache kafka cluster already support the scram-sha-256 and scram-sha-512 user authrization, but kafka operator only support the scram-sha-512. In order to support Kafka more completely, we should also add  scram-sha-256.

**Comment by scholzj** [comment, marked as answer]
I don't think we plan to add support for it right now. It would be relatively lot of work to add it and to maintain it with only a little benefit given we already support SCRAM-SHA-512.
